rust-runtime/aws-smithy-http-server/src/plugin/mod.rs

```diff
@@ -118,6 +118,7 @@
 //! ```
 //!
 
+pub mod check_health;
```

Comment:
I'm hesitant to put it here but I have no better suggestions. Maybe we should think about making a separate crate dedicated to middleware at some point soon perhaps.

Reply:
Yes, I think we'll end up moving these to a separate crate eventually.